### DISCUSSION TITLE
Unable to run foxglove as wss endpoint on cloud

### DISCUSSION THREAD

**Discussion by saumilsdk** [foxglove/community#23] [Q&A]
I have deployed foxglove studio on cloud and exposed https with SSL cert. How can i enable it with wss so that i can connect to rosbridge over websocket locally?

**Comment by jtbandes** [comment]
The comment in https://github.com/foxglove/studio/issues/3125#issuecomment-1086713178 describes the limitations of websocket connections in browsers. Beyond that, the setup may be specific to your use case and platform, and it's hard to provide any advice with the amount of information you've given in the question
